nevergrad/optimization/test_optimizerlib.py

```diff
@@ -581,6 +581,14 @@ def test_shiwa_dim1() -> None:
     assert recom.value < init
 
 
+continuous_case: tp.List[tp.Tuple[str, object, int, int, str]] = [
+    ("NGOpt", d, b, n, f"continuouschoice_d{d}_b{b}_n{n} ")
+    for d in [1, 2, 10, 100, 1000]
+    for b in [2 * d, 10 * d, 100 * d]
+    for n in [1, d, 10 * d]
+]
```

Comment:
can you explain this? I dont really like that information are encoded in a string

Reply:
I am checking that in all those cases we have something reasonable (no SQP or Cobyla if parallel, and one-shot methods when high num workers).

Reply:
I feel f"continuouschoice_d{d}_b{b}_n{n} " is a bit useless here, maybe just make it clearer that it's a hack, like just naming it "#CONTINUOUS"